AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -2767,7 +2767,7 @@ hallhuber.com#$#.cookie-box-modal { display: none !important; }
 balthasar-ress.de###cookieBackdrop
 dinamalar.com##.quickslide
 juwel-aquarium.de,keramundo-onlineshop.de##.cookienote
-geliebtes-zuhause.de,eurotops.de,carlroth.com,hannovermesse.de,loberon.de,appelrath.com,emero.de,sonono.de,reuter.de###econda-pp2-banner
+deerberg.*,geliebtes-zuhause.de,eurotops.de,carlroth.com,hannovermesse.de,loberon.de,appelrath.com,emero.de,sonono.de,reuter.de###econda-pp2-banner
```

Comment:
It was fixed https://github.com/AdguardTeam/AdguardFilters/commit/ee34b691b40b4fe133ce012a111ff484163bef60.
Besides, why do you need `*`?

Reply:
@zloyden Because there are `deerberg.ch`, `deerberg.nl`, `deerberg.be`, `deerberg.at` subdomains.

Reply:
Thanks for the explanation.